src/index.js

```diff
@@ -202,7 +220,7 @@ export default class OasisApp {
       }
       const mcuVersion = tmp.toString();
 
-      return {
+      const info = {
```

Comment:
can you add a comment that typescript goes awry without this

Reply:
Added
```
      // Note: must save to variable before returning, so that inferred type is
      // `{error_message} | {mcuVersion}` instead of `{error_message, mcuVersion?}`.
      // https://github.com/oasisprotocol/ledger-js/pull/415#discussion_r833791641
```